packages/react/src/molecules/pickers/select/base.tsx

```diff
@@ -114,6 +115,8 @@ const PickerSelectBase = React.forwardRef(
             <LabelBase {...props.labelBaseProps} />
           )}
 
+          {props.customRightComponent && <props.customRightComponent />}
+
           {props.clearComponent !== undefined &&
```

Comment:
@Subbu-betalectic Need to also add customRightComponent to these conditions, else the UI might jerk